rpc/src/v1/traits/parity.rs

```diff
@@ -185,5 +185,25 @@ build_rpc_trait! {
 		/// Get the current chain status.
 		#[rpc(name = "parity_chainStatus")]
 		fn chain_status(&self) -> Result<ChainStatus, Error>;
+
+		/// Create new vault.
+		#[rpc(name = "parity_newVault")]
+		fn create_vault(&self, String, String) -> Result<bool, Error>;
```

Comment:
I think those methods should be part of `parity_accounts` namespace (so that they are not expoed to dapps by default)